python/lsst/meas/extensions/shapeHSM/_hsm_shape.py

```diff
@@ -308,18 +324,10 @@ def fail(self, record, error=None):
 class HsmShapeBjConfig(HsmShapeConfig):
     """Configuration for HSM shape measurement for the BJ estimator."""
 
-    def setDefaults(self):
-        super().setDefaults()
-        self.shearType = "BJ"
-
-    def validate(self):
-        if self.shearType != "BJ":
-            raise pexConfig.FieldValidationError(
-                self.__class__.shearType,
-                self,
-                f"shearType '{self.shearType}' is not valid. It must be 'BJ'.",
-            )
-        super().validate()
+    @HsmShapeConfig.shearType.getter
+    def shearType(self):
+        # Docstring inherited.
```

Comment:
I am not so sure that the docstring will be inherited for property.  Running `help(HsmShapeBjConfig.shearType)` gets the docstring, but `HsmShapeBjConfig.shearType.__doc__` does not. And I don't know how `sphinx` will try to get it, and we can take it on in a different issue.

Reply:
Opened a new ticket for this: https://rubinobs.atlassian.net/browse/DM-45784